### PR TITLE
Additional icon partly displaying on https://www.mozilla.org/en-US/firefox/mobile/#focus-privacy #6484

### PR DESCRIPTION
**Pull request by Shivansh2407**
## Description
Additional icon partly displaying on https://www.mozilla.org/en-US/firefox/mobile/#focus-privacy #6484

### CODE REVIEW

File: media/css/firefox/mobile.scss [hunk 1]
```diff
@@ -844,7 +844,7 @@ html[dir="rtl"] {
         }
 
         &.feature-block:before {
-            background-position: right top -4px;
+            background-position: left top -10px;
```
Comment (alexgibson) [inline media/css/firefox/mobile.scss]: This fixes the icon bleed for right-to-left languages, but the icon is no longer aligned correctly:

Screenshot 2019-09-16 at 10 50 27

I'd suggest changing this to:

```suggestion
            background-position: right top -10px;
```

**Comment by Shivansh2407** [comment]
I have committed the requested change . Please let me know if it works well .

**Comment by Shivansh2407** [comment]
Got the left-to-right languages code changed .Please check now whether it got fixed or not?

**Comment by Shivansh2407** [comment]
Your welcome Sir . Will be really glad to contribute more to mozilla .

**Pull request merged**